### PR TITLE
Fixed unnecessary type conversion in dump.

### PR DESCRIPTION
**Pull request by juleq**
Handles quoting STRINGs and ENUMs correctly now.

### CODE REVIEW

**Comment by juleq** [comment]
Fixes #47.

**Comment by juleq** [comment]
Ready to merge when you are :)

File: tests/test_database.py [hunk 1]
```diff
@@ -16,9 +17,12 @@
 except ImportError:
     from io import StringIO
 
+import sys
+sys.path.append('/mnt/d/JQU/dbc/cantools')
```
Comment (eerimoq) [inline tests/test_database.py]: Remove.

File: tests/test_database.py [hunk 2]
```diff
@@ -16,9 +17,12 @@
...
 import cantools
 
 
+
```
Comment (eerimoq) [inline tests/test_database.py]: Remove.

Comment (juleq) [reply tests/test_database.py]: Whoops.

**Pull request merged**